src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -1,25 +1,25 @@
 import * as React from 'react';
 import { ISvgProps, Svg } from '../svg/Svg';
 import * as _ from 'underscore';
-import * as s from 'underscore.string';
 import * as classNames from 'classnames';
 import { FilterBox } from '../filterBox/FilterBox';
 import { keyCode } from '../../utils/InputUtils';
-import { FixedQueue } from '../../utils/FixedQueue';
+import * as s from 'underscore.string';
 
 export interface IDropdownOption {
   svg?: ISvgProps;
   value: string;
   displayValue?: string;
   prefix?: string;
+  selected?: boolean;
+  custom?: boolean;
+  hidden?: boolean;
```

Comment:
quick notice, it seems like hidden and selected follows the same logic everywhere in your code, so it seems like duplication. Instead of using hidden where you use it, use !selected insteand, would it make sens ?